lib/matplotlib/offsetbox.py

```diff
@@ -1031,35 +1041,60 @@ def __init__(self, loc,
                  bbox_transform=None,
                  **kwargs):
         """
-        loc is a string or an integer specifying the legend location.
-        The valid location codes are::
+        Parameters
+        ----------
+        loc : str
+            The box location.
+
+            For back-compatibility, ``'center right'`` (but no other location)
```

Comment:
I think one should just list the valid string positions here (perhaps even ignoring "right") and then say "corresponding numeric locations documented in legend() are also supported" -- otherwise it's really a lot of text that distracts from the main point.